actors/common/src/main/java/org/sunbird/validator/user/UserBulkMigrationRequestValidator.java

```diff
@@ -111,7 +114,7 @@ private void checkEmail(String email,int index){
         if(!isEmailValid){
             errorDetails.setErrorEnum(ErrorEnum.invalid);
         }
-        if(isEmailValid) {
+        if(isEmailValid && EMAIL_DUPLICATE_CHECK) {
```

Comment:
remove this code :         if(isEmailValid && EMAIL_DUPLICATE_CHECK) { } // will never run as of now.

Reply:
@manzarul done